environment/prod/deployment/prod/.env.prod

```diff
@@ -1,4 +1,4 @@
 APP_NAME=Legnicki Meetup Technologiczny
 ENVIRONMENT=prod
 
-LMT_HOST_NAME=`meetup.legnica.pl` , `www.meetup.legnica.pl`
+LMT_HOST_NAME=`meetup.legnica.pl`,`www.meetup.legnica.pl`
```

Comment:
This redirect should be done in the domain DNS zone (www.meetup.legnica.pl -> meetup.legnica.pl).

Reply:
So this is will be the right version?
`LMT_HOST_NAME=meetup.legnica.pl`

Reply:
For me - yes. If we need to support `www.` prefix, then it should be done in the domain DNS zone as CNAME record.